src/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './mysass.scss';
+import PersonalInfo from './PersonalInfo';
+
+class App extends React.Component {
+ 
+
+  render() {
+    return (
+      <div>
+        <div className="style">
```

Comment:
Never name a class as `style`. Its a very ambiguous name.

This could be named `container` or `content` or `header`(i like this one)